### PR TITLE
adding new reminder pack codes

### PR DESCRIPTION
**Pull request by apolloakora**
# Motivation and Context
Adding two new reminder pack codes

# What has changed
Two new pack codes have been added mainly as enums throughout the software.

### CODE REVIEW

**Comment by apolloakora** [comment]
[Add reminder pack codes (3)](https://trello.com/c/6qsTSOf5/1118-add-reminder-pack-codes-3)

**Comment by apolloakora** [comment]
> Code needs formatting (run `mvn fmt:fmt`)

Got it - mvn fmt:fmt for Java and flake for python

**Comment by NickGrantONS** [comment]
> Missing codes `P_RL_2RL2B`, `P_RL_2RL1`. These were already added in the reminder batch scheduler but weren't ever put in the action scheduler/print service

Yep... my findings agree:
```
P_RL_2RL1 - MISSING
P_RL_2RL2B - MISSING
```

**Comment by apolloakora** [comment]
I've added some new codes here including these two codes that you highlighted.
```
P_RL_2RL1 - MISSING
P_RL_2RL2B - MISSING
```

File: src/main/java/uk/gov/ons/census/action/model/entity/ActionType.java [hunk 1]
```diff
@@ -68,12 +72,18 @@
   P_RL_1RL1A(ActionHandler.PRINTER),
   P_RL_1RL2BA(ActionHandler.PRINTER),
   P_RL_2RL1A(ActionHandler.PRINTER),
+  P_RL_2RL2B(ActionHandler.PRINTER),
```
Comment (AdamHawtin) [inline src/main/java/uk/gov/ons/census/action/model/entity/ActionType.java:75]: `P_RL_2RL2B` Isn't a response driven reminder.

**Pull request merged**